addons/l10n_ro/data/account.account.template.csv

```diff
@@ -1,477 +1,477 @@
-"pcg_8036","Redevențe, locații de gestiune, chirii și alte datorii asimilate","8036","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_8037","Efecte scontate neajunse la scadență","8037","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_8038","Bunuri primite în administrare, concesiune și cu chirie","8038","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_8039","Alte valori în afara bilanțului","8039","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_804","Certificate verzi","804","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_8051","Dobânzi de plătit","8051","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_8052","Dobânzi de încasat","8052","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_806","Certificate de emisii de gaze cu efect de seră","806","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_807","Active contingente","807","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_808","Datorii contingente","808","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_809","Creanțe preluate prin cesionare","809","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_891","Bilanț de deschidere","891","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_892","Bilanț de închidere","892","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_901","Decontări interne privind cheltuielile","901","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_902","Decontări interne privind producția obținută","902","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_903","Decontări interne privind diferențele de preț","903","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_921","Cheltuielile activității de bază","921","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_922","Cheltuielile activităților auxiliare","922","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_923","Cheltuieli indirecte de producție","923","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_924","Cheltuieli generale de administrație","924","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_925","Cheltuieli de desfacere","925","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_931","Costul producției obținute","931","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
-"pcg_933","Costul producției în curs de execuție","933","account.data_account_off_sheet","l10n_ro.ro_chart_template","False"
+id,name,code,user_type_id/id,chart_template_id/id,reconcile
```

Comment:
Why all this diff for such a small change?
Please keep the same csv format as before

Reply:
@william-andre i will check it